<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generator</class>
 <widget class="QWidget" name="Generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Generator {
	background-color: white;
}

QLabel {
	color: red;
	font-size: 16px;
	font-weight: bold;
}

QPushButton {
	min-width: 100px;
	min-height: 35px;
	color: white;
	font-weight: bold;
	background-color: red;
	border-radius: 5px;
}

QPushButton:hover:!pressed {
	background-color: #EC3812;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="students_label">
       <property name="text">
        <string>Students:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="teachers_label">
       <property name="text">
        <string>Teachers:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="courses_label">
       <property name="text">
        <string>Courses:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="min_enrolment_label">
       <property name="text">
        <string>Minimum Enrolment:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="max_enrolment_label">
       <property name="text">
        <string>Maximum Enrolment:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="students_input"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="teachers_input"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="courses_input"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="min_enrolment_input"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="max_entolment_input"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generate_button">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
